packages/cli/src/controllers/users.controller.ts

```diff
@@ -551,6 +564,16 @@ export class UsersController {
 	@Post('/:id/reinvite')
 	async reinviteUser(req: UserRequest.Reinvite) {
 		const { id: idToReinvite } = req.params;
+		const isWithinUsersQuota = Container.get(License).isWithinUsersLimit();
+
+		if (!isWithinUsersQuota) {
+			this.logger.debug(
+				'Request to send email invite(s) to user(s) failed because the user limit quota has been reached',
+			);
+			throw new BadRequestError(
```

Comment:
Shouldn't a quota reached error be 403 Forbidden? Request is properly formed (not 400) and authentication succeeded (not 401), but member in Starter has no permission to use this instance.

Reply:
Using 403 everywhere